seatunnel-e2e/seatunnel-spark-connector-v2-e2e/src/test/java/org/apache/seatunnel/e2e/spark/SparkContainer.java

```diff
@@ -97,7 +97,8 @@ public Container.ExecResult executeSeaTunnelSparkJob(String confFile) throws IOE
         // Running IT use cases under Windows requires replacing \ with /
         String conf = targetConfInContainer.replaceAll("\\\\", "/");
         final List<String> command = new ArrayList<>();
-        command.add(Paths.get(SEATUNNEL_HOME, "bin", SEATUNNEL_SPARK_BIN).toString());
+        String sparkBinPath = Paths.get(SEATUNNEL_HOME, "bin", SEATUNNEL_SPARK_BIN).toString();
+        command.add(sparkBinPath.replaceAll("\\\\", "/"));
```

Comment:
Could you please move `replaceAll` method out of `command.add()`? This will make the process clearer.